cumulusci/core/tests/test_config.py

```diff
@@ -210,20 +248,328 @@ def get_service(self, name):
+
+    def test_use_sentry(self):
+        config = BaseProjectConfig(BaseGlobalConfig())
+        config.keychain = mock.Mock()
+        self.assertTrue(config.use_sentry)
+
+    @mock.patch('raven.Client')
+    def test_init_sentry(self, raven_client):
+        config = BaseProjectConfig(BaseGlobalConfig())
+        config.keychain = mock.Mock()
+        config.init_sentry()
+        self.assertEqual(
+            {'repo', 'commit', 'cci version', 'branch'},
+            set(raven_client.call_args[1]['tags'].keys()),
+        )
+
+    @mock.patch('cumulusci.core.config.BaseProjectConfig._get_github_api')
+    def test_get_github_api(self, get_github_api):
+        config = BaseProjectConfig(BaseGlobalConfig())
+        config.keychain = mock.Mock()
+        config.get_github_api()
+        get_github_api.assert_called_once()
+
+    def test_get_latest_version(self):
+        config = BaseProjectConfig(BaseGlobalConfig(), {
+            'project': {
+                'git': {
+                    'prefix_beta': 'beta/',
+                    'prefix_release': 'release/',
+                }
+            }
+        })
+        config.get_github_api = DummyGithub
+        result = config.get_latest_version(beta=True)
+        self.assertEqual('1.0 (Beta 1)', result)
+
+    def test_config_project_path_no_repo_root(self):
+        config = BaseProjectConfig(BaseGlobalConfig())
+        with temporary_dir() as d:
+            cwd = os.getcwd()
+            os.chdir(d)
+            self.assertIsNone(config.config_project_path)
+            os.chdir(cwd)
+
+    def test_get_tag_for_version(self):
+        config = BaseProjectConfig(BaseGlobalConfig(), {
+            'project': {
+                'git': {
+                    'prefix_release': 'release/',
+                }
+            }
+        })
+        self.assertEqual('release/1.0', config.get_tag_for_version('1.0'))
+
+    def test_get_tag_for_version_beta(self):
+        config = BaseProjectConfig(BaseGlobalConfig(), {
+            'project': {
+                'git': {
+                    'prefix_beta': 'beta/',
+                }
+            }
+        })
+        self.assertEqual('beta/1.0-Beta_1', config.get_tag_for_version('1.0 (Beta 1)'))
+
+    def test_get_version_for_tag(self):
+        config = BaseProjectConfig(BaseGlobalConfig(), {
+            'project': {
+                'git': {
+                    'prefix_beta': 'beta/',
+                    'prefix_release': 'release/',
+                }
+            }
+        })
+        self.assertEqual('1.0', config.get_version_for_tag('release/1.0'))
+
+    def test_check_keychain(self):
+        config = BaseProjectConfig(BaseGlobalConfig())
+        with self.assertRaises(KeychainNotFound):
+            config._check_keychain()
+
+    def test_list_orgs(self):
+        config = BaseProjectConfig(BaseGlobalConfig())
+        config.keychain = mock.Mock()
+        config.keychain.list_orgs.return_value = _marker = object()
```

Comment:
should use `unittest.mock.sentinel` for clarity on what we're testing

Reply:
Nice, I didn't know that one.